bagri-server/bagri-server-hazelcast/src/main/java/com/bagri/server/hazelcast/task/doc/DocumentRemoveProcessor.java

```diff
@@ -25,13 +24,13 @@
 
 @SpringAware
 public class DocumentRemoveProcessor implements EntryProcessor<DocumentKey, Document>, EntryBackupProcessor<DocumentKey, Document>, Offloadable {
-    private transient DataDistributionService ddSvc;
-    private transient DocumentManagementImpl docMgr;
 
     private static final long serialVersionUID = 1L;
 
-	private transient DataDistributionService ddSvc;
-    
+    private transient DataDistributionService ddSvc;
```

Comment:
no need for this service. write all business logic in some method in DocumentManagementImpl and call it from this processor

Reply:
done